spec/decorators/step_decorator_spec.rb

```diff
@@ -1,4 +1,22 @@
 describe StepDecorator do
+  describe "#label" do
+    context "when the step is an approval" do
+      it "fetches the correct role text" do
+        user = build_stubbed(:user)
+        step = Steps::Approval.new(user: user).decorate
+        expect(step.label).to eq "Approval"
```

Comment:
good question. I was following the existing specs which did not use the `I18n` value.

I will change all of them.